index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav class="menu-wrap">
+      <div class="logo">
+        <span class="brand">Saw Mon Han</span>
+        <input type="checkbox" class="toggler" />
+        <div class="hamburger"><div></div></div>
+        <div class="menu">
+          <div class="overlay">
+            <div>
+              <ul>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+    <main>
+      <article class="intro">
+        <h1 class="pb branding">
+          I'm Saw Mon,
+          <br />
+          Glad to See you!
+        </h1>
+        <p class="pb introduction">
+          I am a software developer! I can help you build a product, feature or
+          website look through some of your work and experience! If you like
+          what you see and have a project you need coded, don't hesitate to
+          contact me.
+        </p>
+        <p class="pb connect">LET'S CONNECT</p>
+        <p class="pb">
+          <a class="icons" href="mailto:sawmon71293@gmail.com">
+            <ion-icon name="mail-outline"></ion-icon>
+          </a>
+          <a class="icons" href="https://twitter.com/sawmon34268255">
+            <ion-icon name="logo-twitter"></ion-icon>
+          </a>
+          <a class="icons" href="https://github.com/sawmon71293">
+            <ion-icon name="logo-github"></ion-icon>
+          </a>
+          <a class="icons" href="https://www.linkedin.com/in/saw-mon-han/">
+            <ion-icon name="logo-linkedin"></ion-icon>
+          </a>
```

Comment:
- [ ] Kindly, it would be better also to list the icons using a ```<ul>``` tag to group them, and ```<li>``` per icon. 👍 
  
   _Remember The ```<ul>``` tag in HTML is used to create an unordered list, which is a list of items preceded by bullet points. This tag is useful for creating lists of items that do not need to be in a specific order, such as a list of ingredients for a recipe or a list of items to pack for a trip. It helps to create a clear structure and hierarchy on the webpage and make it more readable to the user. 👍 👍_ 
 
- [ ] I think it would be better to adjust the size and the space between your icons to replicate the Figma design 👍 

## Your design
![image](https://user-images.githubusercontent.com/102757126/212694108-b07392dc-3bc3-4d1f-973d-99ec6738f82d.png)

## Figma design
![image](https://user-images.githubusercontent.com/102757126/212694186-028cd635-5211-40a8-aeb5-06f3a3a94f0a.png)